wallet/loader.go

```diff
@@ -100,6 +100,25 @@ func (l *Loader) RunAfterLoad(fn func(*Wallet)) {
 func (l *Loader) CreateNewWallet(pubPassphrase, privPassphrase, seed []byte,
 	bday time.Time) (*Wallet, error) {
 
+	return l.createNewWalletInternal(
+		pubPassphrase, privPassphrase, seed, bday, false,
+	)
+}
+
+// CreateNewWatchingOnlyWallet creates a new wallet using the provided
+// public passphrase.  No seed or private passphrase may be provided
+// since the wallet is watching-only.
+func (l *Loader) CreateNewWatchingOnlyWallet(pubPassphrase []byte,
+	bday time.Time) (*Wallet, error) {
+
+	return l.createNewWalletInternal(
+		pubPassphrase, nil, nil, bday, true,
+	)
+}
+
+func (l *Loader) createNewWalletInternal(pubPassphrase, privPassphrase,
```

Comment:
nit: This can be called just lowercase `createNewWallet` as not exporting it basically means the same as "internal".

Reply:
Done.